README.md

```
@@ -2,313 +2,1523 @@
+- [Adham Dannaway](https://www.adhamdannaway.com) [Ux/Ui Designer & Frontend Developer]
+- [Adil Shamim](https://adilshamim.me) [Machine Learning Engineer | Data Scientist]
+- [Adithya Krishnan](https://www.adithyakrishnan.com)
+- [Aditi Arya](https://aditiarya.netlify.app) [Software Engineer]
+- [Aditya Chaudhary](https://aditya-portfolio-dusky.vercel.app)
+- [Aditya Dubey](https://adityacodes.in)
+- [Aditya Dutt Pandey](https://www.adpandey.com) [Founder | Backend Engineer | System Architect]
+- [Aditya Kumar Gupta](https://aditya30051993.github.io/my-portfolio) [Doctor & Developer]
+- [Aditya Kumar Srivastava](https://adityasri.in) [Full Stack Developer (Mern & Java Springboot Developer)]
+- [Aditya Kumar](https://www.adityakr.com)
+- [Aditya Medhe](https://aditya.medhe.in)
+- [Aditya Pillai] (https://aditya-pillai.vercel.app/) [Computer Science Undergraduate | SWE Intern]
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)